code/datums/components/blur_protection.dm

```diff
@@ -0,0 +1,38 @@
+/datum/component/blur_protection
+	var/mob/living/protected_mob
+
```

Comment:
this could easily be an element

Reply:
i have no idea what the distinction between these is

Reply:
/datum/element vs /datum/component

Datum element creates one instance for all uses
component creates an instance every time it's added